featurebyte/service/tile_manager.py

```diff
@@ -403,12 +413,12 @@ async def remove_tile_jobs(
         TileScheduleNotSupportedError
             if task manager is not initialized
         """
-        if not self._task_manager:
+        if not self.task_manager:
```

Comment:
not really related to this PR, but i wonder if some of these dependency checks can be simplified with by injecting them in now too

Reply:
yes. would be nice to make use of app_container or sth similar as well

Reply:
This is already using dependency injection so this check should be redundant now. Will remove in a next PR.